google/cloud/bigtable/internal/instance_admin.h

```diff
@@ -178,6 +178,57 @@ class InstanceAdmin {
       bigtable::InstanceId const& instance_id,
       bigtable::ClusterId const& cluster_id, grpc::Status& status);
 
+  /**
+   * Makes an asynchronous request to get the attributes of a cluster.
+   *
+   * @param instance_id the id of the instance in the project, from that cluster
```

Comment:
Maybe `s/, from that cluster/from where the cluster is to/`?  Or simply:

```
  * @param instance_id the Cloud Bigtable instance that contains the cluster.
  * @param cluster_id fetch the attributes for this cluster id.  The full name of the
  *      cluster is `projects/<PROJECT_ID>/instances/<instance_id>/clusters/<cluster_id>`
```

Check that last path, because I just might be wrong about how it is formed.